boto/s3/connection.py

```diff
@@ -135,17 +139,36 @@ def build_url_base(self, connection, protocol, server, bucket, key=''):
         return url_base
 
 
-class Location(object):
+class MetaLocation(type):
+    # Metaclass for Location
+    # Allows to access unexisting members of Location without having an
+    # instance of Location.
+    def __init__(self, name, bases, attrs):
+        super(MetaLocation, self).__init__(name, bases, attrs)
+        self.__regions = {
+            # For up to date information, see:
+            # http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region
+            'DEFAULT':      '',  # US Classic Region
+            'EU':           'eu-west-1',
+            'EUWest':       'eu-west-1',
```

Comment:
It's using different keys, with a similar value. That's nothing like a bug, it's the desired behavior.